### PR TITLE
Resource name enhancement

### PR DESCRIPTION
**Pull request by shreekarSS**
ocs/constants.py
  - Added component names for rbd and cephfs

tests/helpers.py
  - Added a variable 'component' in functions create_pvc(),create_storage_class() and create_secret() which passed to create_unique_resource_name() which creates resources.
  - Resources can be identified easily with this component name in the resource name
Signed-off-by: Shreekar <sshreeka@redhat.com>

### CODE REVIEW

File: tests/helpers.py [hunk 1]
```diff
@@ -110,14 +112,17 @@ def create_secret(interface_type):
...
         secret_data = defaults.CSI_CEPHFS_SECRET.copy()
         del secret_data['data']['userID']
         del secret_data['data']['userKey']
         secret_data['data']['adminID'] = constants.ADMIN_BASE64
         secret_data['data']['adminKey'] = get_admin_key()
+        component = constants.CEPHFS_COMPONENT
+
     secret_data['metadata']['name'] = create_unique_resource_name(
-        'test', 'secret'
+        f'test-{component}', 'secret'
```
Comment (ebenahar) [inline tests/helpers.py]: do we need the second arg 'secret'? I think we can remove this arg from create_unique_resource_name()

Comment (shreekarSS) [reply tests/helpers.py]: The second arg 'secret' is    'resource_type' in create_unique_resource_name() which returns  return f"{resource_type}-{resource_description[:23]}-{current_date_time[:10]}"

If we remove second arg then, we can't identify which resource type are we creating

Comment (ebenahar) [reply tests/helpers.py]: yes, I meant that in create_uniques_resource_name, to return:
```python
return f"{resource_description[:23]}-{current_date_time[:10]}"
```

Comment (ebenahar) [reply tests/helpers.py]: Please disregard my above comment

**Pull request closed without merging**